vllm/sequence.py

```diff
@@ -621,6 +621,22 @@ def __repr__(self) -> str:
                 f"num_seqs={len(self.seqs_dict)})")
 
 
+class SequenceGroupMetadataDecode:
```

Comment:
probably should refactor seq group metadata to distinguish scheduler request and in-worker state